datalad/local/foreach_dataset.py

```diff
@@ -167,9 +170,11 @@ class ForEachDataset(Interface):
             `contains` is used"""),
         output_streams=Parameter(
             args=("--output-streams", "--o-s"),
-            constraints=EnsureChoice('capture', 'pass-through'),
-            doc="""whether to capture and return outputs from 'cmd' in the record ('stdout', 'stderr') or
-            just 'pass-through' to the screen (and thus absent from returned record)."""),
+            constraints=EnsureChoice('capture', 'pass-through', 'relpath'),
+            doc="""ways to handle outputs. 'capture' and return outputs from 'cmd' in the record ('stdout',
+            'stderr'); 'pass-through' to the screen (and thus absent from returned record); prefix with 'relpath'
+            and just pass-through (similar to like grep does), and if `dataset` is specified - relative to top
+            of that dataset, if not - to current directory."""),
```

Comment:
@bpoldrack, re your suggestion

> It's a prefixed capture + dedicated rendering to spit it out again (after execution in a ds finished).

what about 

```suggestion
            'stderr'); 'pass-through' to the screen (and thus absent from returned record); prefix with 'relpath'
            captured output (similar to like grep does) and write to stdout and stderr. In 'relpath', relative path
            is relative to the top of the dataset if `dataset` is specified, and if not - relative to current directory."""),
```

Reply:
Yes, fine with me.